WordPress/openverse
# Remove `gevent`, `django-braces`, and `ipaddress` dependencies

## Fixes



Fixes #2787  by @sarayourfriend 
Fixes #2889 by @sarayourfriend 
Fixes #2890 by @sarayourfriend 

## Description


Removes the dependencies referenced in the linked issues.

## Testing Instructions

CI should pass. Run the API locally and confirm it works. Note that gevent would only have been useful if we had been running the Gevent gunicorn worker, which we are not. We may have needed it as a dependency of grequests, which we used to use for dead link validation before we switched to `aiohttp`. But again, we don't use grequests anymore, so we shouldn't need this either.

The other dependencies have zero references in the project, so 

## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
